packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -71,7 +72,7 @@ const DefaultHeader = ({
             if (config.link_to === routes.mt5) {
                 return !is_logged_in || is_mt5_allowed;
             }
-            if (is_mf && config.href === routes.smarttrader) {
+            if ((is_mf || is_options_blocked) && config.href === routes.smarttrader) {
```

Comment:
is_restricted_country?

and is_options_blocked = is_mf || is_restricted_country